website/docs/components/rich-tooltip/partials/guidelines/overview.md

```diff
@@ -0,0 +1 @@
+A Rich Tooltip displays extra supplemental information about a UI element. It's similar to the "standard" [Tooltip](/components/tooltip) but is intended to be used for more complex or structured content instead of to simple text. More details are outlined in [Rich Tooltip vs. Tooltip](/#rich-tooltip-vs-tooltip).
```

Comment:
```suggestion
A Rich Tooltip displays extra supplemental information about a UI element. It's similar to the "standard" [Tooltip](/components/tooltip) but is intended to be used for more complex or structured content instead of simple text. More details are outlined in [Rich Tooltip vs. Tooltip](/#rich-tooltip-vs-tooltip).
```